src/lightning_app/cli/lightning_cli_remove.py

```diff
@@ -0,0 +1,17 @@
+import click
+
+from lightning_app.cli.cmd_ssh_keys import SSHKeyManager
+
+
+@click.group("remove")
+def cli_remove() -> None:
+    """Remove Lightning AI self-managed resources (ssh-keys, etc…)"""
+    pass
+
+
+@cli_remove.command("ssh-key")
+@click.argument("key_id")
```

Comment:
for consistency with the other command, we should call this key_name here too

Reply:
we don't delete by key name, but by key id. id is the pk we generate on the backend.